Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -809,6 +809,7 @@ static ErrorOr<void> generate_unicode_data_implementation(Core::Stream::Buffered
 #include <AK/StringView.h>
 #include <LibUnicode/CharacterTypes.h>
 #include <LibUnicode/UnicodeData.h>
+#include <LibUnicode/Normalize.h>
```

Comment:
This commit should be squashed into the first commit. It's considered a fixup, which we actively avoid within a PR (see [here](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy)).

Reply:
Right, so you would add Normalize.h and Normalize.cpp in the first commit as well.

Reply:
Both now merged into [c8c4aaa](https://github.com/SerenityOS/serenity/pull/15464/commits/c8c4aaafdadafc4b1fdac80a3d469073f09b3686)